File: lsclient.client/src/app/models/CompanyPayload.ts
export class CompanyPayload {
    companyId?:string;
    companyTinNumber?: string;
    companyName?: string;
    adminFullName?: string;
    adminEmail?: string;
    adminContact?: string;
    companyAddress?: string;
    companyDescription?: string;
    companyLatitude?: number;
    companyLongitude?: number;

    constructor(
        companyId?: string,
        companyTinNumber?: string,
        companyName?: string,
        adminFullName?: string,
        adminEmail?: string,
        adminContact?: string,
        companyAddress?: string,
        companyDescription?: string,
        companyLatitude?: number,
        companyLongitude?: number
    ) {
        this.companyId=companyId;
        this.companyTinNumber = companyTinNumber;
        this.companyName = companyName;
        this.adminFullName = adminFullName;
        this.adminEmail = adminEmail;
        this.adminContact = adminContact;
        this.companyAddress = companyAddress;
        this.companyDescription = companyDescription;
        this.companyLatitude = companyLatitude;
        this.companyLongitude = companyLongitude;
    }
}

File: lsclient.Server/Models/DataPayloads/CompanyPayload.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace lsclient.Server.Models.DataPayloads
{
    public class CompanyPayload
    {
        public string? CompanyID { get; set; }

        public string? CompanyTinNumber { get; set; }
             public string? CompanyName { get; set; }
             public string? AdminFullName { get; set; }

             public string? AdminEmail { get; set; }
             public string? AdminContact { get; set; }
             public string? CompanyAddress { get; set; }
             public string? CompanyDescription { get; set; }
             public double? CompanyLatitude { get; set; }
             public double? CompanyLongitude { get; set; }
          

    }
}
